### PR TITLE
Strong params part 2

### PR DESCRIPTION
**Pull request by meara**
https://pm.tablexi.com/issues/140237

### CODE REVIEW

File: app/controllers/account_users_controller.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def new
   def create
     ## TODO add security
     @user                    = User.find(params[:user_id])
-    @account_user            = @account.account_users.new(params[:account_user])
+    @account_user            = @account.account_users.new(create_params)
```
Comment (jhanggi) [inline app/controllers/account_users_controller.rb]: Can we rubocop this and get rid of the extra spaces?

File: app/controllers/bundle_products_controller.rb [hunk 2]
```diff
@@ -24,7 +24,7 @@ def index
 
   # POST /facilities/:facility_id/bundles/:bundle_id/bundle_products
   def create
-    @bundle_product = @bundle.bundle_products.new(params[:bundle_product])
+    @bundle_product = @bundle.bundle_products.new(create_params)
```
Comment (jhanggi) [inline app/controllers/bundle_products_controller.rb]: Does this use `load_and_authorize_resource` if so, can we get rid of this line?

File: app/controllers/facility_facility_accounts_controller.rb [hunk 3]
```diff
@@ -26,7 +26,7 @@ def new
 
   # POST /facilities/:facility_id/facility_accounts(.:format)
   def create
-    @facility_account = current_facility.facility_accounts.new(params[:facility_account])
+    @facility_account = current_facility.facility_accounts.new(facility_facility_account_params)
```
Comment (jhanggi) [inline app/controllers/facility_facility_accounts_controller.rb]: Can we use `load_and_authorize_resource through: :current_facility` and then get rid of this line and the `find` in update?

Comment (meara) [reply app/controllers/facility_facility_accounts_controller.rb]: I can't get that to work--I think it's confused about the class name, despite it being passed explicitly in the `load_and_authorize_resource`. It looks for `facility_facility_account` on current_facility.

**Pull request merged**